app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (NotWearingMaskError,
+                        NotVaccinatedError,
+                        OutdatedVaccineError)
+
+
+def go_to_cafe(friends, cafe) -> str:
+    total = 0
+
+    for i in range(len(friends)):
+        try:
+            cafe.visit_cafe(friends[i])
+        except (NotVaccinatedError, OutdatedVaccineError):
+            return 'All friends should be vaccinated'
+        except NotWearingMaskError:
+            total += 1
+
+    if total:
+        return f'Friends should buy {total} masks'
+    else:
+        return f'Friends can go to {cafe.name}'
```

Comment:
Instead of using an else statement to return the Welcome message, you can simplify the code by directly returning the welcome message after the conditions are checked. This makes the code more concise and easier to read.

Good example:
```
def go_to_cinema(condition: bool) -> str:
    if condition:
        return "We are going to the cinema!"
    return "We are not going to the cinema("
```